src/menu/custom_menu.c

```diff
@@ -41,7 +42,7 @@ static void menu_main_draw_strings(void) {
 
 static void host_menu_draw_strings(void) {
     #ifdef DISCORD_SDK
-        #define HOST_MENU_MAX_ITEMS 4
+        #define HOST_MENU_MAX_ITEMS 5
     #else
         #define HOST_MENU_MAX_ITEMS 3
```

Comment:
this needs to be changed to `HOST_MENU_MAX_ITEMS 4`

Reply:
The text doesn't generate if I set it to four. Do I need a line outside of the ifdef setting `HOST_MEN...``` to 4?

Reply:
I just misinterpreted the feedback. All good now!